src/main/scala/cromwell/engine/workflow/WorkflowOptions.scala

```diff
@@ -0,0 +1,89 @@
+package cromwell.engine.workflow
+
+import com.typesafe.config.ConfigFactory
+import cromwell.util.{EncryptedBytes, SecretKey, Aes256Cbc}
+import scala.collection.JavaConverters._
+import spray.json._
+import scala.util.{Try, Success, Failure}
+
+object WorkflowOptions {
```

Comment:
Right now they are simple key/value pairs.  The reason why there's a `.fromJsonObject(obj: JsObject)`, is that we need to be able to construct workflow options from JSON that looks like the encrypted example above, like if restart the server and we need to reload what's in the database back into a `WorkflowOptions` object.

There's logic in here that says that WorkflowOptions can only be constructed from `JsObject` if the values are either `JsString` or a `JsObject` that looks like `{"iv": "...", "ciphertext": "..."}`